certstrap/tasks/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -o errexit -o nounset -o xtrace
+
+export GOPATH=$PWD/go
+
+package="github.com/square/certstrap"
+go build -ldflags='-s -w' "${package}"
```

Comment:
About a meg (something like 2.9M with the args, 4.3 without).  Explosions seem unlikely (it's ldflags), but I can take them out if you really don't like them.